prototype_source/pt2e_quant_ptq_static.rst

```diff
@@ -0,0 +1,399 @@
+(prototype) PyTorch 2.0 Export Post Training Static Quantization
+================================================================
+**Author**: `Jerry Zhang <https://github.com/jerryzh168>`_
+
+This tutorial introduces the steps to do post training static quantization in graph mode based on
+`torch._export.export <https://pytorch.org/docs/main/export.html>`_. Compared to `FX Graph Mode Quantization <https://pytorch.org/tutorials/prototype/fx_graph_mode_ptq_static.html>`, this flow is expected to have significantly higher model coverage (`88% on 14K models <https://github.com/pytorch/pytorch/issues/93667#issuecomment-1601171596>`), better programmability, and a simplified UX.
+
+Exportable by `torch._export.export` is a prerequisite to use the flow, you can find what are the constructs that's supported in `Export DB <https://pytorch.org/docs/main/generated/exportdb/index.html>`.
+
+tldr; The PyTorch 2.0 Export Quantization API looks like the following:
+
+.. code:: python
+
+  import torch
+  class M(torch.nn.Module):
+     def __init__(self):
+        super().__init__()
+        self.linear = torch.nn.Linear(5, 10)
+
+     def forward(self, x):
+        return self.linear(x)
+
+
+  example_inputs = (torch.randn(1, 5),)
+  m = M().eval()
+
+  # Step 1. program capture
+  m = torch._dynamo.export(m, *example_inputs, aten_graph=True)
+  # we get a model with aten ops
+
+
+  # Step 2. quantization
+  from torch.ao.quantization.quantize_pt2e import (
+    prepare_pt2e,
+    convert_pt2e,
+  )
+
+  from torch.ao.quantization.qnnpack_quantizer import (
+    QNNPackQuantizer,
+    get_symmetric_quantization_config,
+  )
+  # backend developer will write their own Quantizer and expose methods to allow users to express how they
+  # want the model to be quantized
+  quantizer = QNNPackQuantizer().set_global(get_symmetric_quantization_config())
+  m = prepare_pt2e(m, quantizer)
+
+  # calibration omitted
+
+  m = convert_pt2e(m)
+  # we have a model with aten ops doing integer computations when possible
+
+1. Motivation of PyTorch 2.0 Export Quantization
```

Comment:
I suggest to rework this section and make it more to the point of limitations in comparison to the new approach. We don't need to go to the details of which teams need to be involved to develop support, etc. and what can be improved. We should just say: our previous approach while works has the following limitations and the the list. I tried to do this below.